roofit/roofit/src/RooLagrangianMorphFunc.cxx

```diff
@@ -775,7 +775,7 @@ void collectHistograms(
       RooArgSet vars;
       vars.add(var);
 
-      RooDataHist *dh = new RooDataHist(histname, histname, vars, hist);
+      RooDataHist *dh = new RooDataHist(histname.View(), histname.View(), vars, hist);
```

Comment:
Why this is needed ? Should not be the conversion String -> string_view automatic?

Reply:
Yes it should, but for some reason it didn't work on Windows 10: https://github.com/root-project/root/pull/8614#issuecomment-875729534 without this change: https://github.com/root-project/root/pull/8614#issuecomment-875729534.

This is not really a RooFit problem, so I'm not sure it it's worth to address it in this PR.

Reply:
This is strange, then this means that some users might have problems using Window10 calling the constructor with TString ?

Reply:
Yes, but only if they build ROOT with the C++14 standard, so it's a really specific problem.